Cleanup and restructure the README page (#500)
winsw/winsw: README.md

> # WinSW: Windows Service Wrapper in less restrictive license

Reviewer: ```suggestion
# Windows Service Wrapper in less restrictive license
```

See the [project manifest](MANIFEST.md).

## Supported platforms

WinSW offers executables for .NET Framework 2.0, 4.0 and 4.6.1.

> It can run on Windows platforms which have these versions of .NET framework installed.

Reviewer: ```suggestion
It can run on Windows platforms which have these versions of .NET Framework installed.
```

> For systems without .NET Framework, the project provides native 64bit and 32bit executables which are based on .NET Core 3.1.

Reviewer: ```suggestion
For systems without .NET Framework, the project provides native 64-bit and 32-bit executables which are based on .NET Core 3.1.
```

More executables can be added upon request.

## Download

WinSW binaries are available on [GitHub Releases](https://github.com/winsw/winsw/releases) and [NuGet](https://www.nuget.org/packages/WinSW/).

Alternative sources:

* [Maven packaging](https://github.com/jenkinsci/winsw-maven-packaging) for executables, hosted by the [Jenkins project](https://jenkins.io/). 
Binaries are available [here](https://repo.jenkins-ci.org/releases/com/sun/winsw/winsw/).

> * [Puppet](./doc/puppetWinSW.md), currently outdated (WinSW 1.x). Binaries are available on [Puppet Forge](https://forge.puppet.com/kenmaglio/winsw)

Author: FTR I created https://github.com/kenmaglio/winsw/issues/9

Reviewer: It's probably better to distribute the binaries on Chocolatey.

Author: Created https://github.com/winsw/winsw/issues/503 for the backlog

See the [project manifest](MANIFEST.md).

## Supported platforms

WinSW offers executables for .NET Framework 2.0, 4.0 and 4.6.1.
It can run on Windows platforms which have these versions of .NET framework installed.
For systems without .NET Framework, the project provides native 64bit and 32bit executables which are based on .NET Core 3.1.

More executables can be added upon request.

## Download

WinSW binaries are available on [GitHub Releases](https://github.com/winsw/winsw/releases) and [NuGet](https://www.nuget.org/packages/WinSW/).

Alternative sources:

* [Maven packaging](https://github.com/jenkinsci/winsw-maven-packaging) for executables, hosted by the [Jenkins project](https://jenkins.io/). 
Binaries are available [here](https://repo.jenkins-ci.org/releases/com/sun/winsw/winsw/).
* [Puppet](./doc/puppetWinSW.md), currently outdated (WinSW 1.x). Binaries are available on [Puppet Forge](https://forge.puppet.com/kenmaglio/winsw)

## Usage

> WinSW is being managed by configuration files: [Main XML configuration f,ile](doc/xmlConfigFile.md) and [EXE configuration file](doc/exeConfigFile.md).

Author: ```suggestion
WinSW is being managed by configuration files: [Main XML configuration file](doc/xmlConfigFile.md) and [EXE configuration file](doc/exeConfigFile.md).
```

* [Developer guide](DEVELOPER.md)

## Contributing

Contributions are welcome!

> No Clontributor Licence Agreement is needed, just submit your pull requests.

Reviewer: ```suggestion
No Contributor License Agreement is needed, just submit your pull requests.
```

> See [the contributor guidelines](./CONTRIBUTING.md) for more information and guidelines.

Reviewer: ```suggestion
See the [contributing guidelines](./CONTRIBUTING.md) for more information.
```
